src/processHistorian.py

```diff
@@ -112,11 +110,6 @@ def __init__(self):
         print("Work threads:")
         print(self.__threads)
 
-        # Eighth step: Subscribe all data and start all threads
-        self._opcua_client.subscribe_all()
-        for thread in self.__threads:
-            thread.start()
```

Comment:
Removing this also makes the buffer push not working

Reply:
That is true and a big bug, thanks!